src/app/page.tsx

```diff
@@ -4,6 +4,8 @@ import { SignOutButton } from "~/components/authButtons";
 import Profile from "~/components/profile";
 import { Button } from "~/components/ui/button";
 import { getServerAuthSession } from "~/server/auth";
+import CardPremium from "~/components/premiumPricingCard";
+import CardStandard from "~/components/standardPricingCard";
```

Comment:
Components should have the same name as their filenames. Change this to `PremiumPricingCard` and `StandardPricingCard`